src/index.ts

```diff
@@ -58,9 +62,10 @@ export const openDatabase = (
         try {
           // console.warn(`[react-native-quick-sqlite], sql: `, sql, ` params: ` , params);
           let response = sqlite.executeSql(options.name, sql, params);
-          // enhance object to allow the sqlite-storage typeorm driver to work
-          response.rows.item = (idx: number) => response.rows._array[idx];
-
+          if(response.rows) {
+            // enhance object to allow the sqlite-storage typeorm driver to work
+            response.rows.item = (idx: number) => response.rows._array[idx];
```

Comment:
use optional chaining `response.rows?.item`

Reply:
Should I use this instead?
```
response.rows!.item = ()...
```
Using "response.rows?." are causing some lint errors.

Reply:
When I try to use the optional chaining, I got this error:

```
src/index.ts: Invalid left-hand side in assignment expression. (88:10)
```

Using "response.rows!.item" fails too, with the system trying to assign the item function to a non existent response.rows object. On my tests, for now, only this check works as expected

```
if (response.rows) {
    ...
}
```

Reply:
I forgot the rows item is an object